File: structure/teste.c
#include "trie.h"

int main(){

   trie_insert("a", 13);
   trie_insert("abracadabra", 12);

   //trie_insert("abacaxi", 3);

   //trie_insert("a tonito", 5);

   char *str = JSONStringSubTree("a");

   printf("%s \n", str);

   return 0;
}

File: binding/teste.py
from trie_py import Trie
import json

trie = Trie();

trie.insert("a", 12);

trie.insert("abracadabra", 13);

stres = trie.JSONStringSubTree("av");

print(stres)

result = json.loads(stres);

print(result)#, result["value"])